packages/data-stores/src/plans/hooks/test/use-pricing-meta-for-grid-plans.ts

```diff
@@ -257,24 +248,30 @@ describe( 'usePricingMetaForGridPlans', () => {
 			planSlug: PLAN_PERSONAL,
 		} ) );
 
-		const useCheckPlanAvailabilityForPurchase = () => {
-			return {
-				[ PLAN_PREMIUM ]: true,
-				[ PLAN_PERSONAL ]: true,
-			};
-		};
+		Plans.useSitePlans.mockImplementation( () => ( {
+			isLoading: false,
+			data: {
+				[ PLAN_BUSINESS ]: {
+					...SITE_PLANS[ PLAN_BUSINESS ],
+					pricing: {
+						...SITE_PLANS[ PLAN_BUSINESS ].pricing,
+						costOverrides: [ { overrideCode: COST_OVERRIDE_REASONS.RECENT_PLAN_PRORATION } ],
```

Comment:
No idea why this test worked previously. This is needed for the condition to pass, but curious if I'm missing something. @aneeshd16 would you mind confirming?

Reply:
This was previously defined in the mock here: https://github.com/Automattic/wp-calypso/blob/8bab616ec6c7f2d8877bb5c16d88c149fd40a21e/packages/data-stores/src/plans/hooks/test/use-pricing-meta-for-grid-plans.ts#L81